modules/virt-template-fields-web.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_templates/virt-creating-vm-template.adoc
+
+// VM wizard includes additional options to VM template wizard
+// Call appropriate attribute in the assembly
+
+The following table outlines the fields when creating a new custom template in the {VirtProductName} web console. Not all fields are editable.
+
+[id="virt-template-fields-web_{context}"]
+= Custom template web console fields
+|===
+|Name |Description
+
+|Details
+a|
+* Name
+* Namespace
+* Labels
+* Annotations
+* Display name
+* Description
+* Workload profile
+* CPU/Memory
+* Boot mode
+* Base template
+* Created at (time)
+* Owner
+* Boot order
+* Boot source
+* Provider
+* Support
+* GPU devices
+* Host devices
+
+|YAML
+|To view the VM custom resource (CR).
+
+|Scheduling
+a|
+* Node selector
+* Tolerations
+* Affinity Rules
+* Descheduler setting
+* Dedicated resources
+* Eviction strategy
+
+|Network Interfaces
+|Add or edit a network interface.
+
+|Disks
+|Add or edit a disk.
+
+|Scripts
+|Edit the `cloud-init` settings.
+
+|Parameters
+|Edit the description and value type for the template parameters.
```

Comment:
Are you sure about these last 2 rows? This is what I found:

- Scripts: cloud-init settings, SSH key, and Sysprep answer files.
- Metadata: Labels and annotations.

That "Parameters" row looks a bit dodgy. Please check it. 

The merged code appears to currently have the [Metadata tab](https://github.com/kubevirt-ui/kubevirt-plugin/blob/main/src/views/catalog/wizard/tabs.tsx).